tests/loadtest/deployer/README.md

```diff
@@ -67,6 +71,11 @@ started as a bootstrap node
 - `TargetMinorBlockTime` refers to CHAINS/CONSENSUS_CONFIG/TARGET_BLOCK_TIME in cluster config that defines the target block interval of each shard
 - `GasLimit` refers to CHAINS/GENESIS/GAS_LIMIT in cluster config that defines the gas limit for a block; note that in-shard transactions uses 50% of the total gas limit in a block
 
+[This sample config in the repo](./deployConfig-sample.json) illustrates how 3 clusters can be deployed to 17 hosts. 
```

Comment:
how 3 clusters with 256 shards (64 chains * 4 shards per chain) can be deployed to 17 hosts.